README.md

```diff
@@ -25,6 +25,10 @@
 - Run `./build.sh`. If it doesn't run, make sure it is executable.
 > To view the program as doc, run : `godoc -http=:6060` and checkout at `http://localhost:6060/pkg/kwoc-backend/`
 
+### Setting up pre-commit hook (optional)
+- Check if `golangci-lint` is installed, if not, install using `go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.53.3`
```

Comment:
Is it necessary to specify a version here?

Reply:
Also please add a link to the `golangci-lint` repo or website.

Reply:
they say master might not be stable, although it's not recommended to install with go install/get anyway. curl might be a better way, although windows users will need to use git bash to use curl command.

Reply:
Just link to the official installation instructions then

Reply:
and remove the go install?

Reply:
Yes